Services/Masterdata/Services.Masterdata.EF/AutomapperProfile.cs

```diff
@@ -20,6 +20,7 @@ public AutomapperProfile()
             .AfterMap(UpdateMetadata);
 
         CreateMap<Entities.Masterdata, MasterdataModel>();
+        CreateMap<Entities.Masterdata, MasterdataTagModel>();
```

Comment:
Doublecheck: You're mapping 2 times from the same source to a different destination. Is that correct?

Reply:
btw: no need to add the namespace in the source entity. Could be just MasterData instead of Entities.MasterData

Reply:
> btw: no need to add the namespace in the source entity. Could be just MasterData instead of Entities.MasterData

it only works with Entities.Masterdata, otherwise an error occures!

Reply:
> Doublecheck: You're mapping 2 times from the same source to a different destination. Is that correct?

yes